stage0/gc.go

```diff
@@ -125,9 +164,20 @@ func ForceMountGC(path, uuid string) error {
 		return fmt.Errorf("error getting mounts for pod %s from mountinfo: %v", uuid, err)
 	}
 
+	for i := len(mnts) - 1; i >= 0; i -= 1 {
+		mnt := mnts[i]
+		if strings.Contains(mnt.mode, "shared") || strings.Contains(mnt.mode, "master") {
```

Comment:
Hopefully we won't ever get a mode like `shared-foo` or `grandmaster` which would match this `if`, supposedly wrongly.